plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/BuildPath.java

```diff
@@ -70,18 +70,14 @@ public static void addMavenLibraries(IProject project, List<Library> libraries,
     // see m2e-core/org.eclipse.m2e.core.ui/src/org/eclipse/m2e/core/ui/internal/actions/AddDependencyAction.java
     // m2e-core/org.eclipse.m2e.core.ui/src/org/eclipse/m2e/core/ui/internal/editing/AddDependencyOperation.java
     
-    // todo use monitor
-    SubMonitor subMonitor = SubMonitor.convert(monitor,
-        Messages.getString("adding.app.engine.libraries"), libraries.size()); //$NON-NLS-1$
-    
     IFile pomFile = project.getFile("pom.xml");
     
     try {
       Pom pom = Pom.parse(pomFile);
       pom.addDependencies(libraries);
     } catch (SAXException | IOException ex) {
-      IStatus status = new Status(IStatus.ERROR,
-          "com.google.cloud.tools.eclipse.appengine.libraries", ex.getMessage(), ex);
+      IStatus status = StatusUtil.error("com.google.cloud.tools.eclipse.appengine.libraries",
```

Comment:
This probably isn't going to work as expected. The first parameter `origin` is either the instance of the class that is the origin for the error or the class itself. So it probably should be either `this` or `this.getClass()` (or `BuildPath.class`, but moving this code another class during a refactor would require changing it)